lib/bom.js

```diff
@@ -682,14 +682,15 @@ type PositionOptions = {
     maximumAge?: number;
 }
 
-declare class AudioContext {
+type AudioContextState = 'suspended' | 'running' | 'closed';
+
+declare class BaseAudioContext {
   currentTime: number;
   destination: AudioDestinationNode;
   listener: AudioListener;
   sampleRate: number;
-  state: any;
+  state: AudioContextState;
   onstatechange: (ev: any) => any;
-  close(): void;
```

Comment:
`BaseAudioContext` doesn't have `close()` in the spec
https://webaudio.github.io/web-audio-api/#BaseAudioContext